pkg/application/app.go

```diff
@@ -310,7 +310,10 @@ func (app *Avalanche) CopyVMBinary(inputFilename string, subnetName string) erro
 		return err
 	}
 	vmPath := app.GetCustomVMPath(subnetName)
-	return os.WriteFile(vmPath, vmBytes, constants.WriteReadReadPerms)
+	if err := os.WriteFile(vmPath, vmBytes, constants.WriteReadReadPerms); err != nil {
+		return err
+	}
+	return os.Chmod(vmPath, constants.DefaultPerms755)
```

Comment:
why not just write the file with these perms? I don't think you need a separate `chmod`